src-json/define.json

```diff
@@ -261,6 +261,11 @@
 		"doc": "The current Haxe version value in SemVer format.",
 		"reserved": true
 	},
+	{
+		"name": "HaxeNext",
+		"define": "haxe-next",
+		"doc": "Enable experimental features that are meant to be released on next Haxe version."
```

Comment:
I wouldn't talk about "experimental features" here, that's too specific. But I don't have any better phrasing right now.

Reply:
Was "Enable warnings/errors that would otherwise only trigger starting next Haxe version." better?
If that flag ends up doing more (I don't even have candidates atm anyway...) in the future we could potentially update its documentation then to reflect that.

Reply:
I think we should talk about "behavior" in some way, but I agree, let's not worry about this too much for now.